identity/credentials_oidc.go

```diff
@@ -50,6 +51,7 @@ func NewCredentialsOIDC(idToken, accessToken, refreshToken, provider, subject st
 				InitialIDToken:      idToken,
 				InitialAccessToken:  accessToken,
 				InitialRefreshToken: refreshToken,
+				Organization:        organization,
```

Comment:
Since this field is new, and can be undefined in the credential object, the credential version should be +1 and there should be a transformater (if needed) of v0 to v2. Check out WebAuthnm, there we have this already.

Reply:
Organization is a new optional field. Why would I need a new version for this?